mizar/dp/mizar/workflows/builtins/networkpolicies/create.py

```diff
@@ -148,13 +149,32 @@ def fill_cidrs_from_directional_traffic_rules(self, data, indexed_policy_name, d
                 else:
                     data["cidrs_map_no_except"][indexed_policy_name].append(rule_item["ipBlock"]["cidr"])
             elif "namespaceSelector" in rule_item and "podSelector" in rule_item:
-                raise NotImplementedError("Not implemented")
-            elif "namespaceSelector" in rule_item:
-                raise NotImplementedError("Not implemented")
+                namespaces = kube_list_namespaces_by_labels(networkpolicy_opr.core_api, rule_item["namespaceSelector"]["matchLabels"])
+                if namespaces is not None:
+                    namespace_set = set()
+                    for namespace in namespaces.items:
+                        namespace_set.add(namespace.metadata.name)
+
+                    pods = kube_list_pods_by_labels(networkpolicy_opr.core_api, rule_item["podSelector"]["matchLabels"])
+                    if pods is not None:
+                        for pod in pods.items:
+                            if pod.metadata.namespace in namespace_set and pod.status.pod_ip is not None:
+                                data["cidrs_map_no_except"][indexed_policy_name].append("{}/32".format(pod.status.pod_ip))
+            elif "namespaceSelector" in rule_item:                
+                namespaces = kube_list_namespaces_by_labels(networkpolicy_opr.core_api, rule_item["namespaceSelector"]["matchLabels"])
+                if namespaces is not None:
+                    for namespace in namespaces.items:
+                        pods = kube_list_pods_by_namespace(networkpolicy_opr.core_api, namespace.metadata.name)
```

Comment:
Not "networkpolicy_opr.core_api". This should be called from networkpolicy object class, and use the network policy object to get core_api:
https://github.com/CentaurusInfra/mizar/blob/a0c8fae4a49734c424540c3dc5d3069cce70f23e/mizar/obj/net.py#L44

Reply:
Resolved as discussed in scrum meeting.